services/brig/src/Brig/API/Public.hs

```diff
@@ -239,6 +240,11 @@ internalEndpointsSwaggerDocsAPI ::
   PortNumber ->
   S.OpenApi ->
   Servant.Server (VersionedSwaggerDocsAPIBase service)
+internalEndpointsSwaggerDocsAPI service examplePort swagger (Just (VersionNumber V6)) =
+  swaggerSchemaUIServer $
+    swagger
+      & adjustSwaggerForInternalEndpoint service examplePort
+      & cleanupSwagger
```

Comment:
I'm looking at the definition for V0..V3 and they are all `emptySwagger`. Is that so because we had no Servant-based Swagger docs for those versions?

Reply:
As just discussedd, the internal API is not versioned, and therefore only the latest stable V5 and the development version V6 point to the internal API docs. 0-5 should point to empty swagger.